### PR TITLE
feat: add new section

### PR DESCRIPTION
**Pull request by AnzhelikaDavidyan**
add new section

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -12,5 +12,9 @@ <h3>Hello</h3>
     <h4>Hello</h4>
     <p>Lorem ipsum dolor sit amet.</p>
     <p>Lorem ipsum dolor sit amet.</p>
+    <div>
+      <h2>Lorem ipsum</h2>
```
Comment (AnzhelikaDavidyan) [inline index.html:16]: ??

File: index.html [hunk 2]
```diff
@@ -12,5 +12,9 @@ <h3>Hello</h3>
...
+      <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Nesciunt illo blanditiis corrupti qui maiores alias expedita ea laudantium commodi cupiditate dicta nobis culpa totam hic consectetur ab, provident voluptatem doloribus?</p>
```
Comment (Roba2424) [inline index.html:17]: oloalsd

**Review by AnzhelikaDavidyan — COMMENTED** [review]
?

File: index.html [hunk 3]
```diff
@@ -12,5 +12,9 @@ <h3>Hello</h3>
     <h4>Hello</h4>
```
Comment (AnzhelikaDavidyan) [inline index.html:12]:
```suggestion
    <h2>Hello</h2>
```

**Review by AnzhelikaDavidyan — COMMENTED** [review]
done

File: index.html (hunk 2 above)
Comment (Roba2424) [inline index.html:17]: asdflaldjflaslfa

**Review by Roba2424 — APPROVED** [review]

**Pull request merged**